sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AspectSentiment.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The {@link AspectSentiment} model.
+ */
+@Immutable
+public final class AspectSentiment {
+    private final String text;
+    private final TextSentiment sentiment;
+    private final SentimentConfidenceScores confidenceScores;
+    private final int length;
+    private final int offset;
+
+    /**
+     * Create an {@link AspectSentiment} model that describes aspect.
+     *
+     * @param text The aspect text property.
+     * @param sentiment The text sentiment label: POSITIVE, NEGATIVE, MIXED. {@link TextSentiment} has
+     * {@code neutral} sentiment type additionally, but aspect sentiment can only be positive, negative, or mixed.
```

Comment:
{@code neutral} This wouldn't be highlighted correctly since it is an enum. Need to add a an enum highlighter,